config/initializers/okcomputer.rb

```diff
@@ -2,15 +2,23 @@
 
 OkComputer.mount_at = false
 
-if Settings.ils.client == 'SymphonyClient' && !Settings.folio_migration
-  # OKComputer check that checks if we have a connection to symws
-  class SymphonyClientCheck < OkComputer::Check
-    def check
-      ping = SymphonyClient.new.ping
+if Settings.folio_graphql.url || Settings.folio.url
+  okapi_uri = URI.parse(Settings.folio.url)
+  OkComputer::Registry.register(
+    'okapi',
+    OkComputer::PingCheck.new(
```

Comment:
requests just used a ping check for symphony, so I presumed it would be okay here too: https://github.com/sul-dlss/sul-requests/blob/d66af8281956bba20b14e44443e0d25057490bcc/config/initializers/okcomputer.rb#L27

I couldn't find a better health check endpoint in Folio.

Reply:
We could potentially use https://s3.amazonaws.com/foliodocs/api/okapi/p/okapi.html#discovery_health_get , if we were given appropriate permissions:
```ruby
client.send(:get_json, '/_/discovery/health')
/Users/jcoyne85/workspace/sul-dlss/mylibrary/app/services/folio_client.rb:274:in `parse_json': Access for user 'app_mylibrary' (03bd2dc0-2b83-5aa8-aadd-10dfe4400a90) requires permission: okapi.discovery.health.get (FolioClient::IlsError)
```

Reply:
Maybe we can just use our `FolioClient#ping?` method? 

https://github.com/sul-dlss/mylibrary/blob/main/app/services/folio_client.rb#L73

it at least makes sure we logged in